uptane-standard.md

```diff
@@ -734,9 +734,9 @@ Each Secondary SHALL store the latest copy of all metadata required for its own
 
 #### Send images to Secondaries {#send_images_primary}
 
-The Primary SHALL send the latest image to each of its associated Secondaries that have storage to receive it.
+The Primary SHALL send the latest image to each of its associated Secondaries that have sufficient storage to receive it.
 
-For Secondaries without storage, the Primary SHOULD wait for a request from the Secondary to stream the new image file to it. The Secondary will send the request once it has verified the metadata sent in the previous step.
+For Secondaries with limited storage, the Primary SHOULD wait for a request from the Secondary to stream the new image file to it. The Secondary will send the request once it has verified the metadata sent in the previous step.
```

Comment:
It might be nice to formally define 'limited storage' as insufficient storage to receive an image.

Reply:
Good point. Look better now?